src/python_testing/TC_IDM_4_2.py

```diff
@@ -244,18 +317,20 @@ async def test_TC_IDM_4_2(self):
             targets=[Clusters.AccessControl.Structs.AccessControlTargetStruct(cluster=Clusters.BasicInformation.id)],
             subjects=[CR2_nodeid])
 
-        self.add_ace_to_dut_acl(CR1, CR2_limited_ace)
+        # Add limited ACE
+        await self.add_ace_to_dut_acl(CR1, CR2_limited_ace, dut_acl_original)
```

Comment:
I think you've run into a situation where CR2 doesn't have access to anything and you're failing for that reason. Should we maybe add a step to confirm that CR2 CAN subscribe to something?